tools/autograd/load_derivatives.py

```diff
@@ -130,8 +130,8 @@ def load_derivatives(
 
 
 @with_native_function
-def cpp_arguments(f: NativeFunction) -> Sequence[Binding]:
-    return CppSignatureGroup.from_native_function(f, method=False).signature.arguments()
+def dispatcher_arguments(f: NativeFunction) -> Sequence[Binding]:
+    return dispatcher.arguments(f.func)
```

Comment:
this logic was just wrong before, is that right?

Reply:
yeah (it would silently try to spit out `TensorOptions` arguments instead of their exploded version, although that doesn't actually affect any existing formulas)